src/drivers/localstorage.js

```diff
@@ -123,7 +124,7 @@
                     value = serializer.deserialize(value);
                 }
 
-                value = iterator(value, key.substring(keyPrefixLength), i + 1);
+                value = iterator(value, key.substring(keyPrefixLength), iterationNumber++);
```

Comment:
Weird! Interesting. Okay, I'll fix that up and try to do it now. I've gotta pack for holiday though, so if I don't do it now I'll get to it tonight :smile: